schemars_derive/src/lib.rs

```diff
@@ -49,6 +49,14 @@ fn derive_json_schema(
 
     let type_name = &cont.ident;
     let (impl_generics, ty_generics, where_clause) = cont.generics.split_for_impl();
+    let mut where_clause = where_clause.cloned();
+    if let Some(bounds) = cont.serde_attrs.ser_bound() {
+        let where_clause = where_clause.get_or_insert_with(|| WhereClause {
+            where_token: <Token![where]>::default(),
+            predicates: Punctuated::new(),
+        });
+        where_clause.predicates.extend(bounds.iter().cloned());
```

Comment:
We probably want to replace the automatically inserted `JsonSchema` bounds if manual bounds are provided...

Reply:
Hm, this isn't quite that simple either, since we still need `JsonSchema` for `schema_name()` when generating the type names.